docs/ClusterSetup.md

```diff
@@ -183,6 +183,9 @@ ledgerDirectories=data/bookkeeper/ledgers
 
 # Point to local ZK quorum
 zkServers=zk1.example.com:2181,zk2.example.com:2181,zk3.example.com:2181
+
+# Change the ledger manager type
+ledgerManagerType=hierarchical
```

Comment:
I think the misunderstanding here is that these instructions are supposed to point to the config file keys that needs to be changed. The text in the block is not a valid complete configuration. 

If you look in the `conf/bookkeeper.conf`, the `ledgerManagerType=hierarchical` line is already there.

Perhaps the text should make more clear to use the `conf/bookkeeper.conf` and edit the specified lines.